ElectronClient/gui/utils/NoteListUtils.js

```diff
@@ -126,9 +126,22 @@ class NoteListUtils {
 						}
 						clipboard.writeText(links.join(' '));
 					},
-				})
+				}),
 			);
 
+			if (noteIds.length <= 1) {
+				menu.append(
+					new MenuItem({
+						label: _('Pin Item'),
+						click: async () => {
+							const newNote = await Note.load(noteIds[0]);
+							newNote.pinned = !newNote.pinned;
+							await Note.save(newNote);
```

Comment:
Does pinning a note change it's updated timestamp?  Is this intentional?

Reply:
Is there some way to update only one field of the note without changing the updated time? It was not intented.

Reply:
Will setting autotimestamp to false be enough here?